dbms/src/Storages/MergeTree/MergeTreeData.cpp

```diff
@@ -2549,6 +2602,22 @@ MergeTreeData::DataPartsVector MergeTreeData::getAllDataPartsVector(MergeTreeDat
     return res;
 }
 
+DiskSpaceMonitor::ReservationPtr MergeTreeData::reserveSpaceForPart(UInt64 expected_size)
+{
+    constexpr UInt64 SIZE_1MB = 1ull << 20u; ///@TODO_IGR ASK Is it OK?
```

Comment:
10mb

Reply:
IMHO something between 0.5-5Mb. I would pick 1 or 2Mb :) 

Motivation:
1)bigger reservations - worse for testing.
2)100 concurrent 1 row inserts will require 1Gb reservation.
 
BTW - change the name of that const to something like RESERVATION_MIN_ESTIMATION_SIZE 

ANd the MAGIC_CONST below to smth like RESERVATION_MULTIPLY_ESTIMATION_FACTOR

Reply:
Also - shifts in constants require some extra 'in-mind' calculation effort while reading the code.

IMHO:
* `1024*1024` is bit clearer,
* `1*MiB` is even better,  
* `1_MiB` is just perfect (with [user-defined literal](https://stackoverflow.com/a/39622579/1555175))